tests/Fixtures/Integration/misc/braces,phpdoc_indent.test

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Integration of fixers: braces,phpdoc_indent.
```

Comment:
yet one of them leave the collection in invalid state. that's why i said fail, not crash